### ISSUE TITLE
Anchor links are not accessible when navigating with keyboard

### ISSUE THREAD

**Issue by Jan-Eimertenbrink** [mittwald/flow#2041]
### Flow Version

latest

### Bug Description

When navigating the Styleguide with the keyboard, the focus disappears because it moves to buttons for hidden anchor links. As a result, it becomes unclear where the current focus is.

### Steps to Reproduce

1. Open the Styleguide.
2. Use the `Tab` key to navigate through the page.
3. Observe how the focus moves to hidden anchor links.

### Expected Behavior

When anchor links are focused via `Tab`, they should be made visible so that the user can clearly see where the focus is.

### Actual Behavior

Focus is hidden

### How much does this bug impact you?

🟡 Moderate – It's annoying but there's a workaround

### Relevant Logs or Error Output

```shell

```

### Browser Version

Version 140.0.7339.133 (Offizieller Build) (arm64)

### Before Submitting

- [x] I have checked that this issue has not already been reported.
- [x] I have provided all necessary information to reproduce the issue.
